pkg-manager/resolve-dependencies/src/resolveDependencies.ts

```diff
@@ -1241,7 +1241,7 @@ async function resolveDependency (
   // In case of leaf dependencies (dependencies that have no prod deps or peer deps),
   // we only ever need to analyze one leaf dep in a graph, so the nodeId can be short and stateless.
   const nodeId = pkgIsLeaf(pkg)
-    ? pkgResponse.body.id
+    ? `>${depPath}>`
```

Comment:
this was resulting in having `splitNodeId` called with `nodeId` not having `>` around it, which became noticeable with the change above